recipes/tempest-remap/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "tempest-remap" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ClimateGlobalChange/tempestremap/archive/v{{ version }}.tar.gz
+  sha256: b4578c2cb101ba091a10dc914e15ac968257f5db27ca78bc9fb5dbd70bce191f
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - openblas
```

Comment:
I think this can be made Independent of the blas implementation using the instructions here:
https://conda-forge.org/docs/maintainer/knowledge_base.html#blas